### ISSUE TITLE
Diagonal motion vi-keys different from other roguelikes

### ISSUE THREAD

**Issue by purpleposeidon** [tomassedovic/dose-response#3]
Here's nethack's help page on the subject:

```
y k u   7 8 9   Move commands:
 \|/     \|/            yuhjklbn: go one step in specified direction
h-.-l   4-.-6           YUHJKLBN: go in specified direction until you
 /|\     /|\                        hit a wall or run into something
b j n   1 2 3           g<dir>:   run in direction <dir> until something
      numberpad                     interesting is seen
```

Dose Response's uses n & m instead of b & n.

**Comment by tomassedovic** [comment]
Ah indeed, sorry! I'll try to fix that asap. Thanks!

**Comment by tomassedovic** [comment]
Ok, fixed! I'll release a new version (0.5.1) later today.
